pkg/kopia/command/storage/filesystem_args.go

```diff
@@ -0,0 +1,21 @@
+package storage
+
+import (
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/kanisterio/kanister/pkg/logsafe"
+)
+
+const (
+	filesystemSubCommand = "filesystem"
+	pathFlag             = "--path"
+	// DefaultFSMountPath is the mount path for the file store PVC on Kopia API server
+	DefaultFSMountPath = "/mnt/data"
+)
```

Comment:
Move exported flags into `command/const.go`

Reply:
Similar issue would be here as well, we'll have to import the parent. Might be a good idea to have a utils package separately.